.eslintrc.js

```diff
@@ -230,6 +224,15 @@ module.exports = {
     'no-underscore-dangle': 0,
     'no-param-reassign': 0,
     'max-classes-per-file': 0,
+    'import/no-cycle': 0, // TODO: Fix this in a later PR
```

Comment:
all the TODO comments should reference a specific jira, otherwise this will never be done

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-396390